CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Unreleased
+  - No longer showing card icons for overridden card types.
```

Comment:
```suggestion
  - Fix issue where supportedCardBrands overrides were incorrectly showing images for hidden card brands
```

Reply:
Slight tweak to language to make it a little more clear to folks outside our team